extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -184,6 +187,21 @@ const LIFECYCLE_STAGE_TO_ANALYTICS_TRIGGER = {
   'crossDomainIframeLoaded': AnalyticsTrigger.AD_IFRAME_LOADED,
 };
 
+/**
+ * The sandboxing flags to use when applying the "sandbox" attribute to ad
+ * iframes. See http://go/mdn/HTML/Element/iframe#attr-sandbox.
+ * @const {!Array<string>}
+ */
+const IFRAME_SANDBOXING_FLAGS = [
```

Comment:
make this private

Reply:
It already is private, I'm not exporting it so it's not available outside of this module.

Reply:
Practice is to append _ and change @const to @private

Reply:
Made a string and exported for tests, so tests can verify the value of the sandbox attribute.